src/leiningen/vizdeps.clj

```diff
@@ -215,9 +216,7 @@
 
   Each :dep has keys :artifact-name, :version, :conflict?, and :highlight?."
   [project options]
-  (let [profiles (if-not (:dev options)
-                   [:user]
-                   [:user :dev])
+  (let [profiles (:profiles options)
         {:keys [:prune highlight focus]} options
```

Comment:
Not sure why :prune is a keyword not a symbol here (not your change, but while you're in the code?)